org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/converter/ZigBeeBaseChannelConverter.java

```diff
@@ -177,6 +177,16 @@ public void initialize(ZigBeeThingHandler thing, Channel channel, ZigBeeCoordina
         this.coordinator = coordinator;
     }
 
+    /**
```

Comment:
Should the class-level documentation (which describes the life cycle of a converter) be updated to reflect this change?

Reply:
Yep - thanks. I'll update.